lms/static/js/spec/edxnotes/plugins/accessibility_spec.js

```diff
@@ -37,7 +37,9 @@ define([
         });
 
         afterEach(function () {
-            _.invoke(Annotator._instances, 'destroy');
+            while (Annotator._instances.length > 0) {
```

Comment:
Do we need to mutate `Annotator._instances`? If not, `_.map()` would do the trick here. We should strongly prefer immutability where possible.